hiplot/test_experiment.py

```diff
@@ -59,6 +59,26 @@ def objective(trial: "optuna.trial.Trial") -> float:
     xp._asdict()
 
 
+def test_from_optuna_multi_objective() -> None:
+
+    def objective(trial: "optuna.trial.Trial") -> float:
```

Comment:
This should make the CI happy :)
(you can test locally with `mypy --strict --implicit-reexport hiplot`, from the root of the git repo)
```suggestion
    def objective(trial: "optuna.trial.Trial") -> tp.Tuple[float, float]:
```

Reply:
Indeed! Thank you for your comment. I fixed it locally, so I noticed your suggestion after updating this PR.